src/include/execution/sql/generic_hash_table.h

```diff
@@ -71,9 +71,10 @@ class GenericHashTable {
   /**
    * Explicitly set the size of the hash table to support at least @em new_size
    * elements with good performance.
+   * @param tracker MemoryTracker
    * @param new_size The expected number of elements to size the table for
    */
-  void SetSize(uint64_t new_size);
+  void SetSize(common::ManagedPointer<MemoryTracker> tracker, uint64_t new_size);
```

Comment:
nit: I prefer to put `tracker` after `new_size`, which is the more important parameter for this function. You can directly refactor the parameter order of the function signature in CLion.